e2e-tests/discovery/run-test.sh

```diff
@@ -15,6 +15,7 @@ docker compose down
 echo "------------------------------------"
 echo "Starting Docker containers..."
 echo "------------------------------------"
+docker compose up --wait nodeA-backend nodeA
 docker compose up --wait nodeB-backend nodeB
```

Comment:
could collapse this 2 into 1, which speeds it up (since they're started in parallel)